docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,466 @@
+---
+title: Getting started with KLT
+description: Get started with the Keptn Lifecycle Toolkit
+weight: 10
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Keptn Lifecycle Toolkit works whether or not you use a GitOps strategy.
+The following is an imperative walkthrough.
+
+## Prerequisites
+
+- A Kubernetes cluster >= 1.24 (we recommend [Kubernetes kind](https://kind.sigs.k8s.io/docs/user/quick-start/))
+  (`kind create cluster`)
+- [Helm](https://helm.io) CLI available
+
+## Objectives
+
+- Install Keptn Lifecycle Toolkit on your cluster
+- Annotate a namespace and deployment to enable Keptn Lifecycle Toolkit
+- View DORA Metrics
+- Install Grafana and Observability tooling to view DORA metrics
+
+## System Overview
+
+By the end of this page, here is what will be built.
+This system will be built in stages.
+
+![system overview](/docs/install/assets/install01.png)
+
+## The Basics: A Deployment, Keptn and DORA Metrics
+
+Let's start with the basics.
+Here is what we will now build.
+
+A deployment will occur.
+Keptn will monitor the deployment and generate:
+
+- An OpenTelemetry trace per deployment
+- DORA metrics
+
+![the basics](/docs/install/assets/install02.png)
+
+Notice though that the metrics and traces have nowhere to go.
+That will be fixed in a subsequent step.
+
+## Step 1: Install Keptn Lifecycle Toolkit
+
+Install the Keptn Lifecycle Toolkit using Helm:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+Keptn will need to know where to send OpenTelemetry traces.
+Of course, Jaeger is not yet installed so traces have nowhere to go (yet),
+but creating this configuration now means the system is preconfigured.
+
+Save this file as `collectorconfig.yaml`:
+
+```yaml
+---
+apiVersion: options.keptn.sh/v1alpha1
+kind: KeptnConfig
+metadata:
+  name: keptnconfig-sample
+  namespace: keptn-lifecycle-toolkit-system
+spec:
+  OTelCollectorUrl: 'jaeger-collector.keptn-lifecycle-toolkit-system.svc.cluster.local:4317'
+  keptnAppCreationRequestTimeoutSeconds: 30
+```
+
+Apply the file and restart KLT to pick up the new config:
+
+```shell
+kubectl apply -f collectorconfig.yaml
+kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system -l control-plane=lifecycle-operator
+kubectl rollout status deployment -n keptn-lifecycle-toolkit-system -l control-plane=lifecycle-operator --watch
+kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system -l component=scheduler
+kubectl rollout status deployment -n keptn-lifecycle-toolkit-system -l component=scheduler --watch
+```
+
+## Create Namespace for Demo Application
+
+Save this file as `namespace.yaml`.
+The annotation means that Keptn Lifecycle Toolkit is active for workloads in this namespace.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: keptndemo
+  annotations:
```

Comment:
It looks like this is one place where using `labels` is appropriate.

Reply:
unfortunately we do not have labels check for the namespace enabled 😢